packages/console/src/hooks/use-logto-admin-user.ts

```diff
@@ -0,0 +1,27 @@
+import type { User } from '@logto/schemas';
+import type { Optional } from '@silverhand/essentials';
+import useSWR from 'swr';
+
+import { adminTenantEndpoint, meApi } from '@/consts';
+
+import type { RequestError } from './use-api';
+import { useStaticApi } from './use-api';
+import useLogtoUserId from './use-logto-user-id';
+import useSwrFetcher from './use-swr-fetcher';
+
+const useLogtoAdminUser = (): [Optional<User>, () => void, boolean] => {
```

Comment:
`useCurrentUser()` is good?

Reply:
I want to emphasis that this hook returns user information in admin tenant, as we are using the `/me` api in it to fetch data from server.